.eslintignore

```diff
@@ -0,0 +1,5 @@
+/node_modules/*
+/dist/*
+.eslintrc.json
```

Comment:
You do not need to include `.json` files because you run eslint vs `.ts` and `.tsx` files

Reply:
Removed the .eslintignore .json files in 2fe809c